packages/slice-machine/components/Warnings/index.js

```diff
@@ -1,73 +1,77 @@
-        >
-          <FiZap /> <Text ml={2}>Warnings</Text>
-        </Heading>
+        <Heading as="h3">Unable to fetch remote slices</Heading>
       </Flex>
-      <Box p={3} sx={{ overflow: "auto" }}>
-        {
-          // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
-          orderedList.map((warning) => {
-            // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-member-access
-            const Component = Renderers[warning.key];
-            if (!Component) {
-              return null;
-            }
-            return (
-              <Component
-                // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
-                key={warning.key}
-                // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
-                errorType={warning.title}
-                {...warning}
-              />
-            );
-          })
-        }
-        {
-          // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
-          Object.entries(configErrors || {}).length ? (
-            // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-assignment
-            <ConfigErrors errors={configErrors} />
-          ) : null
-        }
-      </Box>
-    </Box>
-  );
-};
+    )}
+  >
+    <Text as="p" mt={1}>
+      This error probably means that you are not connected to Prismic or that
+      your `prismic-auth` token is outdated.
+    </Text>
+    <Text as="p">
+      To generate a new token, type `prismic login` in your CLI. This message
+      you should disappear instantly.
+    </Text>
+    <Text as="p" mt={4}>
+      If the problem persists, check that your `sm.json` file points to the
+      right API endpoint.
+      <br />
+      Full error: {errorType}
+    </Text>
+  </Card>
+);
 
-export default Warnings;
+export const NotConnected = () => (
+  <Card
+    bg="background"
+    bodySx={{ p: 3 }}
+    Header={({ radius }) => (
+      <Flex
+        sx={{
+          p: 3,
+          bg: "headSection",
+          alignItems: "center",
+          justifyContent: "space-between",
+          borderTopLeftRadius: radius,
+          borderTopRightRadius: radius,
+          // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+          borderBottom: (t) => `1px solid ${t.colors?.borders}`,
+        }}
+      >
+        <Heading as="h3">You're not logged in</Heading>
+      </Flex>
+    )}
+  >
+    <Text as="p">
+      In order to push your slices to your Shared Slices bucket, you will need
+      to be connected to Prismic. If you haven't already created a project,
+      please <b>run the setup command</b> first:
+      <Text variant="pre">prismic sm --setup</Text>
```

Comment:
the setup command doesn't push slices, it'll push custom-types but that's about it, `--setup` will also ask them to create a new repo, unless the `--existing-repo` command is provided.